### DISCUSSION TITLE
performance testing dataset

### DISCUSSION THREAD

**Discussion by C8504** [google/or-tools#3124] [CP-SAT questions] [answered]
Hi Experts,

Could I know if we have internal datasets to benchmark version to version performance comparison?
If we do have, could you please share with us the performance improvement report?

Thanks a lot!

John

**Comment by lperron** [comment, marked as answer]
we ?

We routinely run benchmarks on the MIPLIB2017, all JSSP, all PSPLIB, WeightedTardiness, 2d packing, last 9 years of minizinc challenge. + some problems we have collected.

There is not much to share. We continuously improves, mostly by small leaps. One change improves dramatically one small family of instances. These days, I am mostly working on dual strategies (improving the lower bound of the objective). This does not close new instances, but reduces the gap.
